spec/lexers/xml_spec.rb

```diff
@@ -29,6 +29,14 @@
     it 'guesses by source' do
       assert_guess :source => '<?xml version="1.0" encoding="utf-8"?>'
       assert_guess :source => %{#{bom}<?xml version="1.0" encoding="utf-8"?>}
+      assert_guess :source => <<-eos
```

Comment:
Could this assertion be made simpler? Perhaps put the tags on one line and drop the `<jacoco:report>` tag (and maybe the closing tags, too—I don't think they're necessary)?

Reply:
Done!